### ISSUE TITLE
genereated_plugin_registrant

### ISSUE THREAD

**Issue by florian-lefebvre** [Bhat015/optimize_battery#1]
Hi, first of all thanks for your package, it's really useful!

However, there is a something quite annoying with the `genereated_plugin_registrant.dart` auto generated file.

When running the app in development, it generates the following:
```dart
//
// Generated file. Do not edit.
//

// ignore_for_file: directives_ordering
// ignore_for_file: lines_longer_than_80_chars

// some other imports
import 'package:optimize_battery/';

import 'package:flutter_web_plugins/flutter_web_plugins.dart';

// ignore: public_member_api_docs
void registerPlugins(Registrar registrar) {
  // some other registrars
  OptimizationBatteryPlugin.registerWith(registrar);
  registrar.registerMessageHandler();
}
```

As you can see, the import is broken and causes an error that does not prevent the app from running on other targets than web but still, it's quite annoying to get this kind of error when not targeting web.

I think you should have a look at other packages which generate the same thing correctcly to get it working. (i.e. SharedPreferences or UrlLauncher)

As a reproduction repo, here is our app: https://github.com/EduWireApps/ynotes

**Comment by Bhat015** [comment]
Hi,
Thanks for raising this issue and giving out pointers for me to explore similar plugins. I will check these out and try to update the plugin soon

**Comment by florian-lefebvre** [comment]
Looks like it has been fixed in 0.0.2, thanks a lot!
